cloud-deployment-guide/editor-and-features.md

```diff
@@ -80,7 +90,7 @@ Migrating from a self-hosted environment to Tiny Cloud is easy. Remove the exist
 Replace the script tag with the following:
 
 ```js
-<script src="{{ site.cdnurl }}?apiKey=your_API_key"></script>
+<script src="{{ site.cdnurl }}" referrerpolicy="origin"></script>
```

Comment:
This one doesn't mention no-api-key's meaning.

Reply:
fixed:  http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-and-features/